### PR TITLE
Add Notes

### PR DESCRIPTION
**Pull request by Gene0315**
The PR adds notes:
1. Four-digit NACA Airfoil
2. Cubic Bezier Curve
3. Linear Least Square
4. Fit Symmetry Four-digit NACA Airfoil with Cubic Bezier Curve

### CODE REVIEW

File: foil.tex [hunk 1]
```diff
@@ -50,6 +50,242 @@ \chapter*{Introduction}
...
+\begin{flushleft}
+
+The cubic Bezier curve is defined by 4 controll points $P_0(X_0, Y_0)$,
+$P_1(X_1, Y_1)$, $P_2(X_2, Y_2)$, and $P_3(X_3, Y_3)$.
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=0.3\textwidth]{cubic_bezier_curve.eps}
+    \caption{The cubic Bezier curve.}
+    \label{fig:cubic_bezier_curve}
+\end{figure}
```
Comment (yungyuc) [inline foil.tex:173]: There is a latex/pstricks package to draw bezier curves: https://ctan.math.illinois.edu/graphics/pstricks/contrib/pst-bezier/doc/pst-bezier-doc.pdf .  Let's update this using pstricks later.

File: foil.tex [hunk 2]
```diff
@@ -21,13 +21,13 @@
...
 %
-Title that Jie-Yin and Jenny need to decide
+Fit Four-digit NACA Airfoil with Cubic Bezier Curve
 %
 }
 
 \author{
 %
-Yung-Yu Chen
+Yung-Yu Chen and Jie-Yin Lin
```
Comment (yungyuc) [inline foil.tex:30]: @Gene0315 , I add you as a co-author.

File: foil.tex [hunk 3]
```diff
@@ -50,6 +50,242 @@ \chapter*{Introduction}
...
+%%
+\chapter{Cubic Bezier Curve}
+% ref: https://en.wikipedia.org/wiki/B%C3%A9zier_curve#Cubic_B%C3%A9zier_curves
+
+%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\begin{flushleft}
+
+The cubic Bezier curve is defined by 4 controll points $P_0(X_0, Y_0)$,
```
Comment (yungyuc) [inline foil.tex:166]: For general multi-dimensional spaces (2D and 3D), $P_{0, 1, 2, 3}$ are vectors.  For simplicity we can use the Cartesian coordinate system to formulate but later we should generalize using a simple vector notation.

File: foil.tex [hunk 4]
```diff
@@ -50,6 +50,242 @@ \chapter*{Introduction}
...
+% figure downloaded from: https://btnseniorproject17.wordpress.com/2017/05/01/more-unity-part-2-path-following-and-bezier-curves/
+
+\bigbreak
+The function $B(t)$ is
+\begin{equation}
+    B(t) = (1-t)^3 P_0 + 3(1-t)^2 t P_1 + 3(1-t) t^2 P_2 + t^3 P_3
+    \label{eq:cbc:der0}
+\end{equation}
+where $0 \leq t \leq 1$. \\
+The derivative of $B(t)$ is
```
Comment (yungyuc) [inline foil.tex:183]: To be precise, it should write as first-order derivative.

File: foil.tex [hunk 5]
```diff
@@ -50,6 +50,242 @@ \chapter*{Introduction}
...
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%
+\chapter{Cubic Bezier Curve}
+% ref: https://en.wikipedia.org/wiki/B%C3%A9zier_curve#Cubic_B%C3%A9zier_curves
+
+%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\begin{flushleft}
```
Comment (yungyuc) [inline foil.tex:164]: It is not good to use manual typesetting commands like `flushleft`, `\bigbreak`, etc., in the manuscript.  But they can be cleaned up in a later PR.

File: foil.tex [hunk 6]
```diff
@@ -50,6 +50,242 @@ \chapter*{Introduction}
...
+    \caption{The cubic Bezier curve.}
+    \label{fig:cubic_bezier_curve}
+\end{figure}
+% figure downloaded from: https://btnseniorproject17.wordpress.com/2017/05/01/more-unity-part-2-path-following-and-bezier-curves/
+
+\bigbreak
+The function $B(t)$ is
+\begin{equation}
+    B(t) = (1-t)^3 P_0 + 3(1-t)^2 t P_1 + 3(1-t) t^2 P_2 + t^3 P_3
+    \label{eq:cbc:der0}
```
Comment (yungyuc) [inline foil.tex:180]: We should use a shorter notation for equation.  In a future PR let's change to use `{e:cbc:der0}` instead of `eq:`.

**Review by yungyuc — APPROVED** [review]
LGTM.

File: foil.tex (hunk 1 above)
Comment (Gene0315) [reply foil.tex:173]: It looks so cool! But it seems that the pstricks package need to be installed first. I will give it a try.

File: foil.tex (hunk 5 above)
Comment (Gene0315) [reply foil.tex:164]: Ok, I will know better next time.

**Pull request merged**